packages/backend-app-api/src/config/config.test.ts

```diff
@@ -17,13 +17,15 @@
 import { loadConfigSchema } from '@backstage/config-loader';
 import { createConfigSecretEnumerator } from './config';
 import { mockServices } from '@backstage/backend-test-utils';
+import path from 'path';
 
 describe('createConfigSecretEnumerator', () => {
   it('should enumerate secrets', async () => {
     const logger = mockServices.logger.mock();
 
     const enumerate = await createConfigSecretEnumerator({
       logger,
+      dir: path.resolve('../../packages/backend'),
```

Comment:
What's the current best way to mock the file system? I remember there being a migration a few months ago to a new library.

Reply:
`createMockDirectory`,[like here](https://github.com/backstage/backstage/blob/488c9c8675d1f6dfb3c2cfcabe04010e9c22e7a1/packages/cli/src/lib/versioning/packages.test.ts#L98)

Reply:
Yeah, easier to work with than a fully mocked fs though 😁 . I think this test might not be easily jest-able, bc of this block,
```
pkgPath = req.resolve(
          `${name}/package.json`,
          parentPath && {
            paths: [parentPath],
          },
);
```
and lack of jest support https://github.com/jestjs/jest/issues/9543. I can take a deeper look, but before I go any further, is this change actually useful? Do we need to check the full workspace schemas for each backend run?